content/src/main/content/jcr_root/etc/clientlibs/acs-commons/rte-rtl-switch/js/RightToLeftKeyboardShortcut.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2016 by codeflair Gmbh
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+/*
+ * implements a keyboard shortcut ctrl-shift-x to switch both the inline and full screen rich text exitor to
+ * right-to-left mode and back.
+ * Author: Ben Zahler, www.codeflair.ch
+ */
+(function ($) {
+
+    var isCtrl = false;
+    var isShift = false;
+
+    function rtekeyUp(e) {
+        // if control or shift key is released, reset the values
+        if (e.which == 17) {
+            isCtrl = false;
+        }
+        if (e.which == 16) {
+            isShift = false;
+        }
+    }
+
+    function rtekeyDown(key, target) {
+		//check if the user is in an rte editor
+        if (!target.hasClass('coral-RichText-editor') && !target.hasClass('is-edited')) {
+            return;
+        }
+        //check if one of the required keys is pressed.
+        if (key == 17) {
+            isCtrl = true;
+        }
+        if (key == 16) {
+            isShift = true;
+        }
+        if (key == 88 && isCtrl && isShift) {
```

Comment:
Can you drop a comment in that 88 => "x" key? I had to double-check just to make sure I understood what it was doing.